source/developers-guide/digital-publishing-elements/index.md

```diff
@@ -357,10 +357,113 @@ public function onContentBannerFilter(\Enlight_Event_EventArgs $args)
 
 Via the event `$args` you can get the return value with `$args->getReturn()`. The return value contains the hole banner data with all layers and elements. You can do some data manipulation and return the data afterwards. For example, you could load some media data, if the user can select media files in your element.
 
+## Data handling with plugin version >= 1.2.0 ##
+With the SwagDigitalPublishing version 1.2.0 a new way of handling your custom elements and their data was implemented.
```

Comment:
Starting with the "SwagDigitalPublishing" version 1.2.0 we introduced a new way to handle your custom elements and their related data.